src/components/multi-action-button/multi-action-button.spec.tsx

```diff
@@ -76,91 +78,140 @@ describe("MultiActionButton", () => {
       container = null;
     });
 
-    describe.each([
-      ["Enter", "Enter"],
-      ["Space", " "],
-      ["ArrowDown", "ArrowDown"],
-    ])("the %s key is pressed", (name, key) => {
-      it("then the first additional button should be focused", () => {
-        simulateBlur(mainButton);
-        wrapper.find(StyledMultiActionButton).simulate("mouseleave");
-        wrapper.update();
-        mainButton.simulate("keydown", { key });
-        jest.runAllTimers();
+    describe("using keyboard to open children container", () => {
+      it.each([
+        ["Enter", "Enter"],
+        ["Space", " "],
+        ["ArrowDown", "ArrowDown"],
+      ])(
+        "the first additional button should be focused when %s key is pressed",
+        (name, key) => {
+          simulateBlur(mainButton);
+          wrapper.find(StyledMultiActionButton).simulate("mouseleave");
+          wrapper.update();
+          mainButton.simulate("keydown", { key });
+          jest.runAllTimers();
 
-        const firstButton = wrapper
-          .find(additionalButtonsSelector)
-          .find(Button)
-          .at(0)
-          .getDOMNode();
-        expect(firstButton).toStrictEqual(document.activeElement);
-      });
+          const firstButton = wrapper
+            .find(additionalButtonsSelector)
+            .find(Button)
+            .at(0)
+            .getDOMNode();
+          expect(firstButton).toStrictEqual(document.activeElement);
+        }
+      );
 
       it("does not open additional buttons if opened already - coverage", () => {
-        mainButton.simulate("keydown", { key });
+        mainButton.simulate("keydown", { key: "Enter" });
         jest.runAllTimers();
       });
+
+      it("does not call setTimeout if key is not Enter, Space or ArrowDown", () => {
```

Comment:
are we testing setTimeout or preventDefault here? The title says setTimeout but the test is actually testing preventDefault.